service/src/main/java/bio/terra/cbas/dependencies/bard/BardService.java

```diff
@@ -1,84 +1,22 @@
 package bio.terra.cbas.dependencies.bard;
 
-import bio.terra.bard.api.DefaultApi;
-import bio.terra.bard.client.ApiClient;
-import bio.terra.bard.model.EventsEvent200Response;
-import bio.terra.bard.model.EventsEventLogRequest;
-import bio.terra.cbas.dependencies.common.HealthCheck;
 import bio.terra.cbas.model.RunSetRequest;
-import bio.terra.cbas.models.GithubMethodDetails;
 import bio.terra.cbas.models.MethodVersion;
 import bio.terra.common.iam.BearerToken;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Optional;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Async;
 import org.springframework.stereotype.Component;
 
 @Component
-public class BardService implements HealthCheck {
+public interface BardService {
```

Comment:
Should this interface still implement `HealthCheck`?

Reply:
I'm not convinced CBAS should care about Bard's health. CBAS should really be fire-and-forgetting to Bard, since it doesn't actually rely on Bard for anything.

Reply:
I'm inclined to take this out if that's okay with you @cjllanwarne. It's interesting TDR doesn't include Bard in its [status check](https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/status/StatusService.java#L45). I wonder what other services do!